_data/_2023_members.yml

```diff
@@ -18,3 +18,10 @@
   image_url: https://devagrawal09.github.io/portfolio/img/dp.jpg
   github: devagrawal09
   website: https://devagrawal09.github.io/portfolio/
+
+
+- name: Mahesh Gowda
+  short_bio: Hello! I'm a creative IT student with considerable experience and motivation. I really like to experiment with APIs and open data.
+  image_url: https://static.wixstatic.com/media/a33a07_3cbc8a76610b4944bd4231c6163c03f0~mv2.jpg/v1/fill/w_668,h_966/gowdam.jpg
+  github: MSG17
+  website: gowdam.com
```

Comment:
Could you make this `http://gowdam.com`. This link will parse relative to the document, so unless jekyll does some special parsing, it will probably take you to http://acmatuc.org/gowdam.com, which is incorrect.

Reply:
Just did that in my latest update, and updated my picture (I don't know why there is no extension in the image link, which is from Slack). Sorry for the late response.